maven/templates/rules.bzl

```diff
@@ -1,3 +1,5 @@
+# Do not edit. deployment_rules_builder.py autogenerates this file from deployment/maven/templates/rules.bzl
+
 #
 # GRAKN.AI - THE KNOWLEDGE GRAPH
 # Copyright (C) 2018 Grakn Labs Ltd
```

Comment:
Linking the comment made in the other PR https://github.com/graknlabs/grakn/pull/4703#discussion_r241356298.

Reply:
@haikalpribadi I'll remove license header for now